jetty-ee9/jetty-ee9-servlet/src/main/java/org/eclipse/jetty/ee9/servlet/DefaultServlet.java

```diff
@@ -404,10 +403,15 @@ private int getInitInt(String name, int dft)
         return dft;
     }
 
+    @Override
+    public Resource newResource(String resource)
+    {
+        return resolve(resource);
+    }
+
     @Override
     public Resource newResource(URI uri)
     {
-        // TODO optimised path for URI?
```

Comment:
Don't you want to keep the TODO? there's still room for improvement here, isn't there?